ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mac-address-table.j2

```diff
@@ -0,0 +1,30 @@
+{% if mac_address_table is arista.avd.defined %}
+
+## MAC Address Table Summary
```

Comment:
Normally we would have a heading called `MAC Address Table` and a sub sections with `MAC Address Table Summary` and `MAC Address Table Configuration`
Also I don't think "management" is the right place for this. I understand that mac move logging could be considered management, but mac table aging is not. I would prefer a main section for this.